### PR TITLE
Updates for plotting and SCETlib corrections

### PR DESCRIPTION
**Pull request by kdlong**

### CODE REVIEW

**Comment by bendavid** [comment]
fix conflicts?

File: wremnants/theory_tools.py [hunk 1]
```diff
@@ -39,6 +45,27 @@ def define_scale_tensor(df):
...
+    return scaleHist
+
+def define_and_make_pdf_hists(df, axes, cols, pdfset="nnpdf31"):
+    # slice 101 elements starting from 0 and clip values at += 10.0
+    pdfName = pdfMap[pdfset]["name"]
+    pdfBranch = pdfMap[pdfset]["branch"]
+    tensorName = f"{pdfName}Weights_tensor"
+    tensorASName = f"{pdfName}ASWeights_tensor"
+
+    df = df.Define(tensorName, f"auto res = wrem::clip_tensor(wrem::vec_to_tensor_t<double, 101>({pdfBranch}), 10.); res = nominal_weight*res; return res;")
```
Comment (bendavid) [inline wremnants/theory_tools.py:59]: eventually the number of variations and maybe the type (hessian/symhessian (+alphaS)) will need to be added to the map I guess, but ok can be done later when other pdf sets are added here.

Comment (kdlong) [reply wremnants/theory_tools.py:59]: YEs, I'm going to look at this very soon, in validating the new pdf weight samples

**Comment by bendavid** [comment]
The last commit has other changes aside from clearing the notebooks, is it intended?

**Pull request merged**